src/app/api/routes/installations.py

```diff
@@ -31,3 +32,13 @@ async def update_installation(payload: InstallationIn, installation_id: int = Pa
 @router.delete("/{installation_id}/", response_model=InstallationOut)
 async def delete_installation(installation_id: int = Path(..., gt=0)):
     return await crud.delete_entry(installations, installation_id)
+
+
+@router.get("/list_devices/{site_id}", response_model=List[int])
+async def get_all_at_given_ts_and_site(timestamp: datetime, site_id: int = Path(..., gt=0)):
+    query = installations.select(installations.c.device_id)
+    query = query.where(installations.c.site_id == site_id)
+    query = query.where(installations.c.start_ts <= timestamp)
+    query = query.where(installations.c.end_ts >= timestamp)
+
+    return await crud.get_custom_entry(installations)
```

Comment:
I'm trying to think about a way to adapt the query filters so that we don't need that extra crud. Perhaps the others will have some insights

Reply:
At first sight, in fetch_all, we could add logical operator as string in query filter dict add have conditions on query.where according to it, but maybe that complicates things too much. (and would be "==" by defautl)

```python
async def fetch_all(table: Table, query_filters: Optional[Dict[str, Any]] = None) -> List[Dict[str, Any]]:
    query = table.select()
    if isinstance(query_filters, dict):
        for query_filter_key, query_filter_value in query_filters.items():
            query = query.where(getattr(table.c, query_filter_key) == query_filter_value)
    return await database.fetch_all(query=query)

Reply:
You suggest to create a Dict¨str: Tuple(Any, Optionnal[Any])) where the Opional[Any] would be a logical operator ? It could work but I fear that by trying to be too generic, we make it unclear what we are doing in the code.

Reply:
I thought about two propositions:
- either we keep the current query_filters that uses the equality operator, and two other arguments `lte_filters` and `gte_filters` for "less than or equal" and "greater than or equal".
- in the query filter dict, if the value is a dict, the key specifies the operator, if not it defaults to equality

Reply:
We should handle this in a follow-up issue, let's make the query and execution right in the decorated route for now.

Considering that this should be `return await crud.get_custom_entry(query)` rather than `return await crud.get_custom_entry(installations)` and that the tests don't fail, I fear we can't rely on the unittests for now